helm/ffc-iacs-namespace/templates/persistent-volume-claim.yaml

```diff
@@ -1,14 +1,16 @@
 {{- include "ffc-helm-library.persistent-volume-claim" (list . "ffc-iacs-namespace.persistent-volume-claim") -}}
 {{- define "ffc-iacs-namespace.persistent-volume-claim" -}}
+{{- if .Values.deployPersistentVolumeClaim -}}
```

Comment:
This line needs to be at the top of the file.  The lines above will create the PVC resource.  The ones below override the default vales in the helm library.

So as is this doesn't stop the resource being created, just prevents it being configured.